docs/sql-ref-syntax-qry-select-orderby.md

```diff
@@ -18,5 +18,121 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+The <code>ORDER BY</code> clause is used to return the result rows in a sorted manner
+in the user specified order. Unlike the <code>SORT BY</code> clause, this clause guarantees
```

Comment:
link to ```SORT BY```?

Reply:
@huaxingao will do it in the finalization pr.

Reply:
OK if you want to take care of that at the end